### PR TITLE
Create legacy version table, address missing versions in Upgrade Runtime

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3036
Resolves https://github.com/astronomer/docs/issues/3037

### CODE REVIEW

**Review by ben-astro — COMMENTED** [review]
Looks good to me!

**Review by iamakashladdha — COMMENTED** [review]
Overall LGTM, For the Legacy Astro Runtime Versions Table. Runtime Version 4 - You want to include (LTS) as as far as i remember it was an LTS version. It was for both the files

File: astro/runtime-version-lifecycle-policy.md [hunk 1]
```diff
@@ -79,16 +79,25 @@ Occasionally, Astronomer might deviate from the defined response policy and back
...
 If you have any questions or concerns, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Legacy Astro Runtime versions
+
+The following table contains all major Runtime releases which are out of support. Astronomer is not obligated to answer support questions regarding these versions.
+
+| Runtime version                                 | Airflow version | Release date     | End of maintenance date |
+| ----------------------------------------------- | --------------- | ---------------- | ----------------------- |
+| [3](runtime-release-notes.md#astro-runtime-300) | 2.1.1           | August 12, 2021  | February 2022           |
+| [4](runtime-release-notes.md#astro-runtime-400) | 2.2             | March 10, 2022   | September 2023          |
```
Comment (jwitz) [inline astro/runtime-version-lifecycle-policy.md:98]:
```suggestion
| [4](runtime-release-notes.md#astro-runtime-400) (LTS)  | 2.2             | March 10, 2022   | September 2023          |
```

File: astro/runtime-version-lifecycle-policy.md [hunk 2]
```diff
@@ -79,16 +79,25 @@ Occasionally, Astronomer might deviate from the defined response policy and back
...
+| [5](runtime-release-notes.md#astro-runtime-500) (LTS) | 2.3             | April 30, 2022     | April 2024              |
+| [6](runtime-release-notes.md#astro-runtime-600) (LTS) | 2.4             | September 19, 2022 | March 2024              |
+| [8](runtime-release-notes.md#astro-runtime-800)       | 2.6             | April 30, 2023     | November 2023           |
+| [9](runtime-release-notes.md#astro-runtime-900) (LTS) | 2.7             | August 18, 2023    | January 2025            |
 
 If you have any questions or concerns, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Legacy Astro Runtime versions
+
+The following table contains all major Runtime releases which are out of support. Astronomer is not obligated to answer support questions regarding these versions.
```
Comment (lzdanski) [inline astro/runtime-version-lifecycle-policy.md:93]:
```suggestion
The following table contains all major Runtime releases that are no longer supported. Astronomer is not obligated to answer support questions regarding these versions.
```

**Review by lzdanski — APPROVED** [review]
There's one `which` that should be a `that` or have a comma.

**Pull request merged**